django/contrib/admin/models.py

```diff
@@ -44,6 +51,53 @@ def log_action(
             change_message=change_message,
         )
 
+    def log_actions(
+        self, user_id, queryset, action_flag, change_message="", *, single_object=False
+    ):
+        """
+        Method to bulk_create multiple LogEntry objects from a given queryset.
+        """
+        # If log_action is overridden in a subclass, call it to maintain
+        # backward compatibility
+        if type(self).log_action != LogEntryManager.log_action:
+            return [
+                self.log_action(
+                    user_id=user_id,
+                    content_type_id=ContentType.objects.get_for_model(
+                        obj, for_concrete_model=False
+                    ).id,
+                    object_id=str(obj.pk),
```

Comment:
Why we're calling `str()` on primary keys? `pk` is passed directly in the previous version.

Reply:
This was there in the earlier function, That's why I kept it. Removing it doesn't effect though. https://github.com/django/django/blob/main/django/contrib/admin/models.py#L41